src/cdomains/intDomain.mli

```diff
@@ -412,10 +412,10 @@ module IntervalSetFunctor(Ints_t : IntOps.IntOps): SOverflow with type int_t = I
 module Interval32 :Y with (* type t = (IntOps.Int64Ops.t * IntOps.Int64Ops.t) option and *) type int_t = IntOps.Int64Ops.t
 
 module BigInt:
-  sig
-    include Printable.S with type t = Z.t (* TODO: why doesn't this have a more useful signature like IntOps.BigIntOps? *)
-    val cast_to: Cil.ikind -> Z.t -> Z.t
-  end
+sig
+  include Printable.S with type t = Z.t (* TODO: why doesn't this have a more useful signature like IntOps.BigIntOps? *)
+  val cast_to: Cil.ikind -> Z.t -> Z.t
+end
 
 module Interval : SOverflow with type int_t = IntOps.BigIntOps.t
 
```

Comment:
whitespace change only, should not be part of PR.

Reply:
For some other files (like this one) it is more complicated. This change has been made by my ocp-indent, and when I try to revert it, I shall not commit because ocp-indent complains about the changes.

Reply:
Should I force the commit anyway?